be/src/exec/pipeline/scan/scan_operator.h

```diff
@@ -61,6 +61,11 @@ class ScanOperator : public SourceOperator {
         return res;
     }
 
+    virtual size_t max_scan_concurrency() const {
+        // It takes effect, only when it is positive.
```

Comment:
For `ConnectorScanOperator`, is there any scan concurrency limitation?

Reply:
No, it doesn't have any limitation for now.